packages/tinybird/src/client.ts

```diff
@@ -4,18 +4,24 @@ import * as z from "zod";
 // REMINDER:
 // const tb = new Tinybird({ token: process.env.TINYBIRD_TOKEN! });
 
+export const tinyBirdEventType = z.object({
+  id: z.string(),
+  workspaceId: z.string(),
+  pageId: z.string(),
+  monitorId: z.string(),
+  timestamp: z.number().int(),
+  statusCode: z.number().int(),
+  latency: z.number().int(), // in ms
+  url: z.string(),
+  metadata: z.string().optional().default(""),
+  region: z.string().min(4).max(4), // TODO: object + stringify json
+});
```

Comment:
We will probably have two different schemas for the `publishPingResponse` (datasource) and `getResponseList` (pipe). 

We don't have to return all the values, could group the entries,.. or transform the values e.g.

```
metadata: z.string().transform((m) => JSON.parse(m))
```
while saving it as string.

Reply:
Sure just extract it when I did some small update